notes/2025/2025-06-01-contributor-pool.md

```diff
@@ -0,0 +1,146 @@
+# Contributor Pool Report for May 2025
+
+## xbinaryx (1)
+1. [feat: automatically assign issues to users willing to submit PRs](https://github.com/eslint/eslint-github-bot/pull/219)  
+   This pull request introduced a plugin that automatically assigns issues to users who express willingness to submit pull requests via a checkbox in the issue template. It aims to streamline issue management and improve contributor engagement. The changes also addressed and closed issue #218.  
```

Comment:
A note that `#218` would be written as as a full link or at least escaped since it's not an issue or PR in this repo, but not sure how to tell AI.

Reply:
Yeah, I think that's asking too much at this point. The important thing is the overall report with the PRs linked. The summary is jut to help us more quickly triage.